lib/PuppeteerSharp/Helpers/DictionaryExtensions.cs

```diff
@@ -1,13 +1,12 @@
 using System.Collections.Generic;
-using System.Linq;
 
 namespace PuppeteerSharp.Helpers
 {
     internal static class DictionaryExtensions
     {
         internal static Dictionary<TKey, TValue> Clone<TKey, TValue>(this Dictionary<TKey, TValue> dic)
         {
-            return dic.ToDictionary(entry => entry.Key, entry => entry.Value);
+            return new(dic, dic.Comparer);
```

Comment:
Using the `Dictionary<TKey, TValue>` copy-constructor has some fast paths where it just copies over the hashed keys and values without any rehashing.